data_sources/sysmon_eventid_11.yml

```diff
@@ -92,6 +92,15 @@ fields:
 - timestartpos
 - user_id
 - vendor_product
+output_fields:
```

Comment:
@P4T12ICK we should add process_name and update the searches/risk_messaage

Reply:
I will fix the integration test failures.